pkg/domain/infra/abi/generate.go

```diff
@@ -124,6 +124,15 @@ func (ic *ContainerEngine) GenerateKube(ctx context.Context, nameOrIDs []string,
 		if err != nil {
 			return nil, err
 		}
+		if len(po.Spec.Volumes) != 0 {
+			warning := `
+# NOTE: If you generated this yaml from an unprivileged and rootless podman container on an SELinux
```

Comment:
Isn't this something we can know? We know if we're rootless, and AFAIK the SELinux package has the ability to check if we're enabled

Reply:
We could check for that and add the `privileged: true` option, but that would be assuming the user wants to run a privileged container. We could add a warning that we made it privileged due to volumes.

Reply:
Well the spec file could be shared with people on a different system with SELinux enabled.

Reply:
Why are we doing this in `podman generate kube`, now that I think about this? This mostly seems like an issue for `podman play kube` - shouldn't we warn people there, given the problems show up when the YAML is run?

Reply:
It shows up when we run the yaml in a kubernetes cluster, so makes more sense to put the warning on the generate side.

Reply:
The note will be added to the generated yaml, so users should see it when using it in a kubernetes cluster or with play kube.